howto/serialize/README.md

```diff
@@ -0,0 +1,64 @@
+# Serialization in Dapr's SDKs
+
+An SDK for Dapr should provide serialization for two use cases. First, for API objects sent through request and response payloads. Second, for objects to be persisted. For both these use cases, a default serialization must be provided. In the Java SDK, it is the [DefaultObjectSerializer](https://dapr.github.io/java-sdk/io/dapr/serializer/DefaultObjectSerializer.html) class, providing JSON serialization.

```

Comment:
There are following things and we should talk about serialization/desalinization for all:
1. State Management,
2. Service Invocation Request/Response.
3. Publishing Events.
4. Bindings.

For Actors:
1. Actor Proxy to Actor Service Communication Request/Response.
2. Actor state management.

Reply:
Done.